app/views/organisation/ContactEmailView.scala.html

```diff
@@ -44,6 +44,7 @@ <h1 class="govuk-heading-l">@messages("contactEmail.heading", contactName)</h1>
                 label = LabelViewModel(messages("contactEmail.heading", contactName)).withCssClass( "govuk-visually-hidden")
             )
             .withWidth(Full)
+            .withAttribute("email" -> "email")
```

Comment:
.asEmail() exists in the class FluentInput, should we use that?

Reply:
https://github.com/hmrc/advance-valuation-rulings-frontend/blob/0fb0f914b1a18bcf0ee1ecd018fbdd8a65d7a570/app/viewmodels/govuk/InputFluency.scala#L59

might be worth adding these methods to our service^